patches/base-path.diff

```diff
@@ -119,7 +119,7 @@ Index: code-server/lib/vscode/src/vs/server/node/webClientServer.ts
 --- code-server.orig/lib/vscode/src/vs/server/node/webClientServer.ts
 +++ code-server/lib/vscode/src/vs/server/node/webClientServer.ts
 @@ -267,12 +267,11 @@ export class WebClientServer {
- 			return res.end();
+ 			return void res.end();
```

Comment:
weird that it now has `void`? I thought didn't realize you could use it like this. Upstream change.

Reply:
Ah yeah I think that makes it ignore the return of the function?  To avoid having to put the return on a separate line.